fahclient/templates/statefulset.yaml

```diff
@@ -36,7 +38,7 @@ spec:
       containers:
       - name: {{ .Chart.Name }}
         securityContext:
-          {{- toYaml .Values.securityContext | nindent 12 }}
+          {{- toYaml .Values.containerSecurityContext | nindent 12 }}
```

Comment:
I think this was a typo?

Reply:
Nope, see the change in `values.yaml` also.  You cannot apply 
````
  capabilities:
    drop:
      - ALL
  readOnlyRootFilesystem: false
```` 
at the spec level, so in order to break this apart I had to provide those values from their own key.

Reply:
ah for some reason I missed the values.yaml change. Github app is terrible. I'm going to test your changes now but you may want to update the README with the new values and bump a minor or hotfix version.